class/DatabaseConnection.php

```diff
@@ -0,0 +1,340 @@
+		return $row;
+	}
+
+	/**
+	* To select the desired eamil from the database
+	*
+	* @access public
+	* @param  integer $id
+	* @return object
+	*/
+	public function select_email($email, $emp_id = 0)
+	{
+		
+		$q_fetch = "SELECT * FROM employee WHERE email = '{$email}'";
+
+		if($emp_id > 0)
+		{
+			$q_fetch .= "id != {$emp_id}";
+		}
+
+		$result = DatabaseConnection::db_query($q_fetch);
+		return $result;
+	}
+
+	/**
+	* To select the desired eamil from the database
+	*
+	* @access public
+	* @param  integer $id
+	* @return object
+	*/
+	public function select_login($email, $password)
+	{
+		$q_fetch = "SELECT * FROM employee WHERE email = '{$email}' AND password = '{$password}'";
+		$result = DatabaseConnection::db_query($q_fetch);
+		return $result;
+	}
+
+	/**
+	* To insert data into the database
+	*
+	* @access public
+	* @param  string $table_name, array $data_array
+	* @return object
+	*/
+	public function insert_full($table_name, $data_array)
+	{
+		if($table_name == 'employee')
+		{
+			$q_employee = "INSERT INTO $table_name (first_name, middle_name, last_name, email, 
+				password, prefix, gender, dob, marital_status, employment, employer, photo, 
+				extra_note, comm_id) 
+				VALUES ('{$data_array['first_name']}', '{$data_array['middle_name']}', 
+				'{$data_array['last_name']}', '{$data_array['email']}', 
+				'{$data_array['password']}', '{$data_array['prefix']}', 
+				'{$data_array['gender']}', '{$data_array['dob']}', 
+				'{$data_array['marital_status']}', '{$data_array['employment']}', 
+				'{$data_array['employer']}', '{$data_array['photo']}', 
+				'{$data_array['extra_note']}', '{$data_array['comm_id']}')";
+
+			$result_employee = DatabaseConnection::db_query($q_employee);
+
+			if(!$result_employee)
+			{
+				log_error('Data insertion in employee table');
+				echo "Data insertion into employee table unsuccessful";
+				exit;
+			}
+
+			return self::$conn;
+		}
+		elseif($table_name == 'address')
+		{
+			$q_address = "INSERT INTO $table_name (`emp_id`, `address_type`, `street`, `city`, 
+				`state`, `zip`, `phone`, `fax`) 
+				VALUES
+				({$data_array['emp_id']}, 'residence', '{$data_array['r_street']}', 
+				'{$data_array['r_city']}', '{$data_array['r_state']}', '{$data_array['r_zip']}', 
+				'{$data_array['r_phone']}', '{$data_array['r_fax']}'),
+
+				({$data_array['emp_id']}, 'office', '{$data_array['o_street']}', 
+				'{$data_array['o_city']}', '{$data_array['o_state']}', '{$data_array['o_zip']}', 
+				'{$data_array['o_phone']}', '{$data_array['o_fax']}')";
+
+			$result_address = DatabaseConnection::db_query($q_address);
+
+			if(!$result_address)
+			{
+				log_error('Data insertion in address table');
+				echo "Data insertion into address table unsuccessful";
+				exit;
+			}
+
+			return self::$conn;
+		}		
+	}
+
+	/**
+	* To delete data from the database
+	*
+	* @access public
+	* @param  string $table_name, integer $id
+	* @return void
+	*/
+	public function delete_from_table($table_name, $id)
```

Comment:
Can rename it delete_employee. In place of calling function twice for different tables.
Can call once to delete employee and delete record from both tables.